sdk/python/kfp/v2/dsl/__init__.py

```diff
@@ -12,4 +12,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from kfp.dsl import *
+from kfp.v2.dsl.component import component
```

Comment:
This looks like a circular reference. `kfp.v2.dsl` is importing from `kfp.v2.dsl`.

Perhaps you can use `from .component import component`

Reply:
I don't think there's a circular reference here.

Reply:
Sorry, I was wrong, fixed the filename.